bessctl/bessctl

```diff
@@ -141,14 +142,15 @@ def main():
         line_buf = []
 
         for arg in sys.argv[1:]:
+            arg = arg.decode()
```

Comment:
Yes, I guess it is easier to just make sure the input are Unicode (already did) instead of handling StringIO and BytesIO separately. I am attempting to avoid extra packages like `six` and `future` to keep the dependency minimal 😄